recipes-domu/domu-image-android/domu-image-android.bbappend

```diff
@@ -1,30 +1,20 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-FILESEXTRAPATHS_prepend := "${THISDIR}/../../recipes-domx:"
+# Most functionality and settings are defiend in xt-images.
+# Only some minimal tuning is required by environment variables.
 
-# we need MACHINEOVERRIDES from DomD build
-do_configure[depends] += "domd-image-weston:do_domd_install_machine_overrides"
+# We use SRC_URI provided by meta-xt-images
 
-SRC_URI = " \
-    repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_devel/domu_android_host_tools.xml;scmdata=keep \
-"
+# Android itself doesn't care about hardware, but graphics does.
+# That's why we need to check that we are building for supported hardware.
+python __anonymous () {
+    machines = d.getVar("MACHINEOVERRIDES", expand=True)
+    if ("r8a7795-es3" not in machines):
+        bb.fatal("Android can be built for H3 (r8a7795) ES3 only. Your MACHINEOVERRIDES: %s." % machines)
+}
 
-XT_BB_LAYERS_FILE = "meta-xt-prod-extra/doc/bblayers.conf.domu-image-android"
-XT_BB_LOCAL_CONF_FILE = "meta-xt-prod-extra/doc/local.conf.domu-image-android"
-
-###############################################################################
-# extra layers and files to be put after Yocto's do_unpack into inner builder
-###############################################################################
-# these will be populated into the inner build system on do_unpack_xt_extras
-XT_QUIRK_UNPACK_SRC_URI += "\
-    file://meta-xt-prod-extra;subdir=repo \
-    file://meta-xt-prod-domx;subdir=repo \
-"
-
-# these layers will be added to bblayers.conf on do_configure
-XT_QUIRK_BB_ADD_LAYER += " \
-    meta-xt-prod-extra \
-    meta-xt-prod-domx \
-"
-
-XT_BB_IMAGE_TARGET = "android"
+# For proper build of graphics we need to set proper SOC.
+# For now android is supported only for H3 ES3 (r8a7795)
+SOC_FAMILY = "r8a7795"
```

Comment:
I think even for now we should think about the future
Why not `?=` at least?

Reply:
Reason is that we support android only on H3 ES3 for now.
Weak default assignment is used in main recipe in xt-images
https://github.com/xen-troops/meta-xt-images/pull/197/commits/102ee8bedb0b1389cfcc68d8b9e972243b838e98#diff-40e21da8563c8353e3008858ea6a70be5b0796c6ee4b209d39aa7348f213bd1aR54

But here in product's recipe, I think it good to have '=' assignment, as only supported value is used.